package.json

```diff
@@ -12,7 +12,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "mongodb": "^6.3.0",
```

Comment:
Consider locking the versions of your dependencies to ensure consistent behavior across different environments. [medium]